docs/pages/admin-guides/management/operations/ca-rotation.mdx

```diff
@@ -267,22 +324,33 @@ $ curl https://example.teleport.sh/.well-known/open-id-configuration | jq '.jwks
 Once you have chosen a CA to rotate and have planned to check or update the
 infrastructure that relies on that CA, you are ready to begin a manual rotation.
 
+<Admonition type="tip">
+In 17.1.0+ `tctl auth rotate` (with no arguments) starts an interactive
```

Comment:
note to self: update this with the actual version where https://github.com/gravitational/teleport/pull/49171 merges